### PR TITLE
add kind ingress

### PR DESCRIPTION
**Pull request by plungu**

### CODE REVIEW

File: kind/config.yaml [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+  extraPortMappings:
+  - containerPort: 80
+    hostPort: 80
+    protocol: TCP
+  - containerPort: 443
+    hostPort: 443
+    protocol: TCP
```
Comment (falko) [inline kind/config.yaml:18]: Shouldn't this be optional, e.g. in a second example file that is only used when using the `camunda-ingress` target?

**Review by falko — CHANGES_REQUESTED** [review]
Looks good so far. I'd only suggest to make the ingress completely optional. Did you test running the values without ingress?

**Comment by falko** [comment]
Depends on https://github.com/camunda-community-hub/camunda-8-helm-profiles/pull/12

File: README.md [hunk 2]
```diff
@@ -289,32 +289,49 @@ experimenting with Kubernetes.
...
 
 Or, try `make port-tasklist`, and then access task list here: http://localhost:8082
 
+NOTE: learn more about ingress-kind https://kind.sigs.k8s.io/docs/user/ingress/#ingress-nginx
+
+## Run Kind with Ingress and full stack
+
+- Run the make command
+
+`make camunda-ingress chartValues=../../camunda-8-helm-profiles/development/camunda-values.yaml,../../camunda-8-helm-profiles/ingress-kind/camunda-values.yaml`
```
Comment (falko) [inline README.md:305]: This would be nice in the Helm profiles readme to show people how to invoke custom targets and combine values files

**Comment by falko** [comment]
@plungu we should get this done before #3 happens

File: kind/config.yaml (hunk 1 above)
Comment (plungu) [reply kind/config.yaml:18]: This doesn't seem to effect the startup of the development/default profile.

**Comment by upgradingdave** [comment]
Merged this into camunda-8-helm-profiles

**Pull request closed without merging**